wireapp/kalium
# feature: connect to upgrade-personal-to-team API #WPB-11992

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Adds calls to the API when migrating from personal to team account.

Needs releases with:

- [api v7](https://github.com/wireapp/kalium/pull/3099)

### Testing

#### Test Coverage (Optional)

Work in Progress


#### How to Test

To test you need to switch to anta 
https://wearezeta.atlassian.net/wiki/spaces/ENGINEERIN/pages/300351887/Testing+environments+for+federation#anta

The html with deeplinks switching can be downloaded here. You must logout first before switching. https://wearezeta.atlassian.net/wiki/spaces/ENGINEERIN/pages/300351887/Testing+environments+for+federation#anta

To get signup code you must read the email from [inbucket](https://inbucket.anta.wire.link/monitor). 
Password is on [1password](https://start.1password.com/open/i?a=FOQLGGWZU5EQXEJO3V3A5HI3DU&v=nqjxjxdabagisaqjlyfym4kye4&i=gprcgudpieokefbatxqlxctu2m&h=wire.1password.eu).

Then:
- create personal account
- go to you profile page, at the top you will see migration banner
- go through the process
- at the you should see success screen
- currently you will only see changes to your account after app restart, WIP: https://wearezeta.atlassian.net/browse/WPB-12187


----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
